R/sem.R

```diff
@@ -1386,15 +1404,23 @@ checkLavaanModel <- function(model, availableVars) {
 
     if (options[["outputStandardizedResiduals"]]) {
       # actually compute the implied covariance
-      sv <- lavaan::residuals(fit, type = "standardized")
-      sr <- sv$cov
-      sr[upper.tri(sr)] <- NA
-
-      for (i in 1:ncol(sr)) {
-        nm <- colnames(sr)[i]
-        srtab$addColumnInfo(nm, title = .unv(nm), type = "number", format = "sf:4;dp:3;p:.001")
+      if (options[["se"]] == "bootstrap") {
+        srtab$setError(gettext("The standardized residual covariance table is currently unavailable when the error calculation method is 'Bootstrap'"))
```

Comment:
alternatively you can just ignore this option and disable the standardized residuals option in QML when `bootstrap` is selected. But this is a fine solution as it is.